tools/workspace-plugin/jest.config.ts

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable */
 export default {
   displayName: 'workspace-plugin',
-  preset: '../../jest.preset.js',
+  // TODO: For some reason our patched jest resolve cannot work with @nx/powerpack-conformance
```

Comment:
Needs some time spent on it, there's a lot of custom complexity there, the default resolver works fine...

Reply:
Looks like this is blocking the PR after all because the existing code in this library needs the custom resolver, damn

Reply:
Found it, the custom resolver special cased all `@nx/` packages, but we don't want it to do that with `@nx/powerpack-` ones, so I adjusted that in https://github.com/nrwl/nx/pull/29078/commits/6bf90638fc40823c39be8c3e3055d2d63e62c4bd